orderbook/openapi.yml

```diff
@@ -561,14 +561,15 @@ components:
           type: string
           enum:
             [
+              WrongOwner,
```

Comment:
What is the purpose of moving this to the top? I kind liked that it was in alphabetical order before.

Reply:
Sorry! I had copied it from the order in the `enum`. I'll change it back to alphabetical order.